controller/users.go

```diff
@@ -197,11 +217,11 @@ func ConvertUser(request *goa.RequestData, identity *account.Identity, user *acc
 		Type: workitem.SimpleType{Kind: workitem.KindString},
 	}
 
-	converted := app.Identity{
-		Data: &app.IdentityData{
+	converted := app.User{
+		Data: &app.UserData{
 			ID:   &id,
 			Type: "identities",
```

Comment:
Type will be "users" because of changes inside https://github.com/almighty/almighty-core/pull/1076/files#diff-5a990e44c9a8391f2956c4b6d0128401R32